diff --git a/dist/esm/server/request/search-params.js b/dist/esm/server/request/search-params.js
index 1eda3eb98a85acbda28d1a5dc053b5ba00ed80b5..2b8f4e95c822e63068520b266e0776f7316a7b26 100644
--- a/dist/esm/server/request/search-params.js
+++ b/dist/esm/server/request/search-params.js
@@ -68,11 +68,8 @@ export function createServerSearchParamsForMetadata(underlyingSearchParams, isRu
 export function createServerSearchParamsForServerPage(underlyingSearchParams, varyParamsAccumulator, isRuntimePrefetchable) {
     const workStore = workAsyncStorage.getStore();
     if (!workStore) {
-        throw Object.defineProperty(new InvariantError('Expected workStore to be initialized'), "__NEXT_ERROR_CODE", {
-            value: "E1068",
-            enumerable: false,
-            configurable: true
-        });
+        // Fallback for /_global-error prerender where workStore is not initialized (Next.js bug E1068)
+        return Promise.resolve({});
     }
     const workUnitStore = workUnitAsyncStorage.getStore();
     if (workUnitStore) {
diff --git a/dist/server/request/search-params.js b/dist/server/request/search-params.js
index 41a8c843928e347abb0f6b56847ccb5af0ae3357..18679be35a36a283a81e410acf1b766d601bac72 100644
--- a/dist/server/request/search-params.js
+++ b/dist/server/request/search-params.js
@@ -101,11 +101,8 @@ function createServerSearchParamsForMetadata(underlyingSearchParams, isRuntimePr
 function createServerSearchParamsForServerPage(underlyingSearchParams, varyParamsAccumulator, isRuntimePrefetchable) {
     const workStore = _workasyncstorageexternal.workAsyncStorage.getStore();
     if (!workStore) {
-        throw Object.defineProperty(new _invarianterror.InvariantError('Expected workStore to be initialized'), "__NEXT_ERROR_CODE", {
-            value: "E1068",
-            enumerable: false,
-            configurable: true
-        });
+        // Fallback for /_global-error prerender where workStore is not initialized (Next.js bug E1068)
+        return Promise.resolve({});
     }
     const workUnitStore = _workunitasyncstorageexternal.workUnitAsyncStorage.getStore();
     if (workUnitStore) {
